modules/osdk-run-deployment.adoc

```diff
@@ -78,7 +131,53 @@ $ oc get deployment -n <project_name>-system
 NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
 <project_name>-controller-manager       1/1     1            1           8m
 ----
+endif::[]
+ifdef::java[]
+[source,terminal]
+----
+$ oc get all -n default
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                      READY   UP-TO-DATE   AVAILABLE   AGE
+pod/memcached-quarkus-operator-operator-7db86ccf58-k4mlm   0/1       Running   0           18s
+----
+
+. Apply the `memcached-sample.yaml` to create the `memcached-sample` pod:
```

Comment:
Clarity suggestion:
```suggestion
. Apply the `memcached-sample.yaml` to create the `memcached-sample` pod with the following command:
```

Reply:
Alternatively: "Run the following command to apply the `memcached-sample.yaml` and create the `memcached-sample` pod:"